### ISSUE TITLE
Weird behaviour when pressing backspace

### ISSUE THREAD

**Issue by trashhead** [Gbuomprisco/ngx-chips#671]
PLEASE MAKE SURE THAT:
- you searched similar issues online (9/10 issues in this repo are solved by googling, so please...do it)
- you provide an online demo I can see without having to replicate your environment
- you help me by debugging your issue, and if you can't, do go on filling out this form

**I'm submitting a ...**  (check one with "x")
```
[x ] bug report => search github for a similar issue or PR before submitting
[ ] support request/question

Notice: feature requests will be ignored, submit a PR if you'd like
```

**Current behavior**

When using following combination of attributes it adds tag when pressing backspace:
<tag-input [(ngModel)]='teman' [removable]='true' addOnBlur="true" separatorKeyCodes="[13,188,190,186]" modelAsStrings="true"></tag-input>

When using following combination of attributes it navigates away from the page when trying to remove tag with backspace:
<tag-input [(ngModel)]='teman2' [removable]='true' addOnBlur="true" modelAsStrings="true"></tag-input>

Separator codes is: Enter, Comma, Period, Semicolon

**Expected behavior**
I would like to be able to erase my text and the tag with backspace.

**Minimal reproduction of the problem with instructions (if applicable)**
Stackblitz is availible here, both examples:
https://stackblitz.com/edit/angular-jvzmvk

**What do you use to build your app?. Please specify the version**
Stackblitz

**Angular version:** 
5.0.0

**ngx-chips version:**
1.6.4 (latest version when typing)

**Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]
Chrome (Both examples above does not work)
Edge (First example doesnt work, second example works!)
IE11 (First example doesnt work, second example works!)

**Comment by trashhead** [comment]
@Gbuomprisco, have you been able to reproduce this ?

**Comment by trashhead** [comment]
Sorry, I found an error I had done. I forgot to add brackets around separatorKeyCodes. So writing [separatorKeyCodes]="[13,188,190,186]" worked alot better. I also added an eventlistener for onRemove and in that eventlistener I ran event.stopPropagation() to get it not to navigate from the page.
Here is a stackblitz: 
https://stackblitz.com/edit/angular-lfe8r7

So the only thing I guess I still think might be a bug is that I have to manually stopPropagation for the navigation not to take place, but thats a minor.
 Thanks for the nice work !
